samples/Azure.Management.Storage/Generated/Models/StorageAccountCheckNameAvailabilityParameters.cs

```diff
@@ -23,12 +23,12 @@ public StorageAccountCheckNameAvailabilityParameters(string name)
             }
 
             Name = name;
-            Type = "Microsoft.Storage/storageAccounts";
+            ResourceType = "Microsoft.Storage/storageAccounts";
         }
 
         /// <summary> The storage account name. </summary>
         public string Name { get; }
         /// <summary> The type of resource, Microsoft.Storage/storageAccounts. </summary>
-        public string Type { get; }
+        public string ResourceType { get; }
```

Comment:
This is the case where an enum is named `Type`, then previous code bug renamed `Type` to `Type`. Therefore it didn't throw exception. And the generated type is string because the enum only has one value and generator transformed it to a string with hard-coded value.